docs/17-porting-tips.md

```diff
@@ -94,9 +94,22 @@ work. Remove this section.
     - end
     --- }}}
 
+The `quit` menu command must be wrapped in a function, otherwise an error
+occurs due to mismatched argument types from the v4.0 `awful.menu` library.
+
+     -- {{{ Menu
+     -- Create a laucher widget and a main menu
+     myawesomemenu = {
+       { "manual", terminal .. " -e man awesome" },
+       { "edit config", editor_cmd .. " " .. awesome.conffile },
+       { "restart", awesome.restart },
+    -  { "quit", awesome.quit }
+    +  { "quit", function() awesome.quit() end}
```

Comment:
I probably confused here.  The diff is an example for the docs, not a patch against the code base.  I don't think any of the diff examples in the porting guide will work with patch, though I didn't try.  When I was updating my `rc.lua` using the porting guide, I applied the diffs by hand.